src/VisualStudio/Setup/VisualStudioSetup.csproj

```diff
@@ -17,121 +17,204 @@
     <ImportVSSDKTargets>true</ImportVSSDKTargets>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <RoslynProjectType>Vsix</RoslynProjectType>
+    <IsProductComponent>true</IsProductComponent>
+    <ExtensionInstallationRoot>CommonExtensions</ExtensionInstallationRoot> 
```

Comment:
Is this value the same for all of our VSIX projects which are IsProductComponent?